README.md

```diff
@@ -103,8 +103,16 @@ handling logic.
 		if envPort := os.Getenv("PORT"); envPort != "" {
 			port = envPort
 		}
-		if err := funcframework.Start(port); err != nil {
-			log.Fatalf("funcframework.Start: %v\n", err)
+		
+		// By default, listen on all interfaces. If testing locally, run with 
+		// LOCAL_ONLY=true to avoid triggering firewall warnings and 
+		// exposing the server outside of your own machine.
+		hostname := ""
+		if localOnly := os.Getenv("LOCAL_ONLY"); localOnly == "true" {
```

Comment:
I wonder if this should be a default feature of functions framework i.e. checked for in Start(port). Documenting it as a sample and making the utilities available to do it also LGTM.

Reply:
I thought about that, but it seemed like signing us up for another flag we'd have to maintain indefinitely and a potential new foot-gun in the library. If this library was shared and could exhibit the same behavior across all functions frameworks I think the calculus might change a bit.